packages/e2e/address.test.js

```diff
@@ -1,18 +1,18 @@
-describe("mocking support", () => {
```

Comment:
Why did you comment this?
Please note the `it` is x'ed (`xid` so the test is skipped anyway)